sidecars/sonarlint/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM quay.io/eclipse/che-sidecar-java:11-7bd8c8c
```

Comment:
you'll update parent to match this réponse side-cars in a next step ?

Reply:
I'm not sure such updates are needed, as the images in the `che-sidecar-*` quay.io repos are still valid. I would say leave them and they can be updated on an ongoing basis, i.e. when the plugin is next updated.

On a side note, I'm wondering if sonarlint even needs its own sidecar if all it installs is nodejs. Maybe we can just put nodejs in the java sidecar?

Reply:
Thinking about it more, the flow where sidecars depend on another sidecar as the base needs more thought. For now it would be a two-step PR process: update base, then update sidecar using that base.

We could probably nest them somehow and adjust the build script to account for it, but it's beyond this PR. :smile: